### PR TITLE
Feature/Issue resolved- Improve table stats readability in UI

### PR DESCRIPTION
**Pull request by ShubhamThakre**
Feature Issue link - https://feature-requests.datahubproject.io/b/User-Experience/p/improve-table-stats-readability-in-ui

Adding screenshot for the stats new changes
image
image

UI developer- Shubham Thakre
Reporting- Gabe Lyons

### CODE REVIEW

File: datahub-web-react/src/app/entity/shared/tabs/Dataset/Stats/snapshot/TableStats.tsx [hunk 1]
```diff
@@ -31,15 +32,14 @@ const StatContainer = styled.div<{ justifyContent }>`
...
     return (
         <StatSection>
             <Typography.Title level={5}>Table Stats</Typography.Title>
             <StatContainer justifyContent={justifyContent}>
                 {rowCount && (
                     <InfoItem title="Rows">
                         <Typography.Text strong style={{ fontSize: 24 }}>
-                            {rowCount}
+                            {countFormatter(rowCount)}
                         </Typography.Text>
```
Comment (gabe-lyons) [inline datahub-web-react/src/app/entity/shared/tabs/Dataset/Stats/snapshot/TableStats.tsx:43]: for this, lets also make it so hovering over the formatted count provides the full number

Comment (gabe-lyons) [reply datahub-web-react/src/app/entity/shared/tabs/Dataset/Stats/snapshot/TableStats.tsx:43]: you can use https://ant.design/components/tooltip/

Comment (ShubhamThakre) [reply datahub-web-react/src/app/entity/shared/tabs/Dataset/Stats/snapshot/TableStats.tsx]: Sure, I have also added the comma separated number for long numbers to view better in tooltip. Specifically showed tooltip over right, as top tooltip is hiding the title and we got enough space in right. 

Updated screen shots; 
image

**Review by shirshanka — APPROVED** [review]
LGTM

**Pull request merged**